### DISCUSSION TITLE
Project kickoff call

### DISCUSSION THREAD

**Discussion by robertgzr** [nodejs/single-executable#19] [General]
Let's have an informal video call to kick off work/planning on this project!
Due to timezones, we will probably have to compromise on a time slot - any suggestions what would work best?

As for an agenda - let's keep it light-weight and use this to come up with some structure for future calls. I would suggest we review the requirements for the subprojects outlined in [the blog post](https://github.com/nodejs/single-executable/blob/main/blog/2022-08-05-an-overview-of-the-current-state.md#the-architecture-of-seas) and try to come up with some milestones.

cc @RaisinTen @dsanders11 

---

# Meeting info

zoom link: https://zoom.us/j/97381983106

## First kickoff call timing

12:30pm UTC - 1:30pm UTC on Monday, 5 September, 2022

## Second kickoff call timing

TODO

**Comment by robertgzr** [comment]
we need to figure out how to have a public call, that allows guests to join (and we should probably have a recording afterwards)

**Reply by RaisinTen** [reply]
I have asked for access to the creds required for streaming the meetings in https://github.com/nodejs/admin/issues/717.

**Comment by RaisinTen** [comment]
+1

> any suggestions what would work best?

I think we could have a Google Spreadsheet where everyone could fill in their availabilities and we could plan this accordingly.

cc @nodejs/single-executable

**Reply by RaisinTen** [reply]
@mhdawson is there a template we can use to create this spreadsheet like the TSC meeting time sheet?

**Reply by robertgzr** [reply]
Alternatively, we collect some proposals and then vote using reaction emojis :)

**Reply by RaisinTen** [reply]
I mean the spreadsheet would be a nice place to enter the proposals and we could visually see the overlaps and choose what's best?

**Comment by ovflowd** [comment]
+1 for the call, excited to meet y'all 😄

**Comment by jviotti** [comment]
@RaisinTen Just to have a plan B, if using Node.js' Zoom account doesn't work for whatever reason, I think we can also piggy-back on Postman's Zoom account

**Reply by RaisinTen** [reply]
I have access to the Node.js Zoom account now :)

**Comment by mhdawson** [comment]
+1 on a call, the spreadsheet would  work but we have also often used a [doodle](https://doodle.com/en/)

**Comment by sheplu** [comment]
+1 on a call. doodle seems to be often used but don't really care which solution we choose

**Comment by RaisinTen** [comment]
@nodejs/single-executable I have created a doodle for this meeting - https://doodle.com/meeting/participate/id/b2vkJGPb. Feel free to select the time slots you all are available in. Suggestions welcome!

**Reply by jviotti** [reply]
Done!

**Reply by ljharb** [reply]
9/5 is a Federal holiday in the US, so it might not be the best choice? Also, those times are prohibitively early for the west coast of the US.

**Reply by RaisinTen** [reply]
@ljharb hmm, should we move it to the week after? Also, any suggestions on a friendlier time slot for the US west coast that we should add to the list?

**Reply by aymen94** [reply]
@RaisinTen I suggest you use this tool to schedule a meeting time multiple timezones [timeanddate](https://www.timeanddate.com/worldclock/meeting.html?iso=20160101&more=4%20target=)

**Reply by ljharb** [reply]
The week after, or just "not the specific Monday" - as for better times, 2, 3, or 4 hours later would likely work better, as would 12-14 hours later.

**Reply by RaisinTen** [reply]
I have extended it by 1 hour for all days, PTAL @nodejs/single-executable. I won't be able to go beyond that because it would get super late for me. :/

**Reply by ruyadorno** [reply]
If it's a limitation on your end @RaisinTen and in case you end up with Doodle results which are too-conflicting. May I suggest as a possible solution, to hold a second kickoff call at an alternative time (friendlier to west-coast timezone)?

From quickly looking up the folks in this group I know there's at least @mhdawson and I who could host the call but there might be even more since this can be quite the popular subject. 😊

**Reply by jviotti** [reply]
Yeah, timezone is definitely an issue here, as the core people doing work in this team range from the west coast of the US all the way to India. The way we make it work at Postman is to pick a time early in the morning PT and in the evening for @RaisinTen, but it depends on the willingness of people in the US for having the call pretty early :P 

If that doesn't work, then having two calls might be the best way to go!

**Reply by jviotti** [reply]
Third option: we move the American continent closer to India, but that will take more time than next week 😆

**Reply by mhdawson** [reply]
@jviotti if you can move continents closer together that woul help the overall timezone issue a lot :)

**Reply by jviotti** [reply]
Haha that will take another Node.js team though :P

**Reply by Qard** [reply]
I'll do my best to attend when I can, but I live in west coast and work with people in Europe, so my mornings are generally already claimed. 😐

**Reply by RaisinTen** [reply]
@ruyadorno 

> If it's a limitation on your end @RaisinTen and in case you end up with Doodle results which are too-conflicting. May I suggest as a possible solution, to hold a second kickoff call at an alternative time (friendlier to west-coast timezone)?

Yes, let's do that!

> From quickly looking up the folks in this group I know there's at least @mhdawson and I who could host the call but there might be even more since this can be quite the popular subject. 😊

Sounds great! 💯

**Reply by RaisinTen** [reply]
@nodejs/single-executable according to the doodle results, it seems that 2:30pm UTC - 3:30pm UTC on Friday, 9 September is the best time slot because it has the highest attendance. However, that will conflict with the existing Node-api team meeting. I've heard from @mhdawson yesterday that we can't use the zoom/stream for 2 meetings at a time, so let's go with the 2nd best time slot - 12:30pm UTC - 1:30pm UTC on Monday, 5 September.

Here's the webinar link in advance - https://zoom.us/j/97381983106

**Reply by ljharb** [reply]
To reiterate, that's 5:30AM on the US west coast (8:30AM on the east coast) on a US federal holiday.

**Reply by ovflowd** [reply]
@RaisinTen this doesn't sound like a nice time for US-based folks 😅

**Reply by bnb** [reply]
I won't be there for the reasons @ljharb mentioned 😅

**Reply by RaisinTen** [reply]
Yes, that's truly unfortunate and I think the best thing we can do is have a second kickoff call at a time that's friendlier to the US-based folks as @ruyadorno said. I would encourage y'all to take up the initiative to arrange for that call ASAP!

**Reply by RaisinTen** [reply]
@nodejs/single-executable hello! Is everyone joining now?

**Reply by arcanis** [reply]
@RaisinTen I joined, but I don't think I can talk 🤔

**Reply by jviotti** [reply]
@RaisinTen I joined now after seeing this message by accident. I think you need to send an e-mail invite, at least for the main people in the call, otherwise most of us would forget!

**Reply by ovflowd** [reply]
Agreed. I also completely forgot the meeting. I think it would be better to reschedule and send meeting invitations.

**Reply by ovflowd** [reply]
I honestly thought our meeting was going to be 9th September, wasn't super clear for me 😅

**Reply by RaisinTen** [reply]
> @RaisinTen I joined, but I don't think I can talk 🤔

@arcanis I guess "promoting to panelist" fixed that :)

**Reply by RaisinTen** [reply]
> @RaisinTen I joined now after seeing this message by accident. I think you need to send an e-mail invite, at least for the main people in the call, otherwise most of us would forget!

> I honestly thought our meeting was going to be 9th September, wasn't super clear for me 😅

@jviotti @ovflowd TBH, that's exactly what was done. A Doodle mail was supposed to be sent to all participants along with calendar invites.

<img width="1154" alt="Screen Shot 2022-09-06 at 4 19 46 PM" src="https://user-images.githubusercontent.com/42526976/188616972-a16f1fff-94d2-4a3f-8271-342467f3c497.png">
